package-builder/flox-build.mk

```diff
@@ -377,6 +394,76 @@ define BUILD_nix_sandbox_template =
 
 endef
 
+# The following template renders targets for the sandbox build mode.
+define BUILD_nix_expression_template =
+
+  $(eval _pvarname = $(subst -,_,$(_pname)))
+
+ 	$(eval _result = result-$(_pname))
+  $(eval $(_pvarname)_tmpBasename = $(TMPDIR)/$($(_pvarname)_shortHash)-$(_pname))
+  $(eval $(_pvarname)_buildMetaJSON = $($(_pvarname)_tmpBasename)-build-meta.json)
+
+  .PHONY: $(_pvarname)_nix_sandbox_build
+  $(_pvarname)_nix_expression_build: $($(_pvarname)_buildScript) $($(_pvarname)_src_tar) \
+		$(if $(_do_buildCache),$($(_pvarname)_buildCache))
```

Comment:
Oooh ... are we carrying forward a build cache for NEF builds? Hadn't expected to do that but why not!?!

Reply:
BuildCache et.al is likely to be a remnant of me copying the build template, not deliberate -- i think they are not quite that relevant to nef builds either

Reply:
FTR I removed all logic pertaining to buildCache in commit fbb9ed791055531000fc6f0a939893f376a725e5